### PR TITLE
Adds Enhanced Ecommerce support to Measurement Protocol Uploader

### PR DESCRIPTION
**Pull request by caiotomazelli**

### CODE REVIEW

File: megalista_dataflow/uploaders/google_analytics/google_analytics_measurement_protocol.py [hunk 1]
```diff
@@ -38,29 +38,62 @@ def start_bundle(self):
...
+    metadata_list = batch.execution.destination.destination_metadata
 
-    # parameters starting with ** are optional.
-    payloads = [{
+    # initialize payload with values that are common to all types of hit
+    payload = {
       "v": 1,
-      "tid": execution.destination.destination_metadata[0],
-      "ni": execution.destination.destination_metadata[1],
-      "t": "event",
```
Comment (Mr-Lopes) [inline megalista_dataflow/uploaders/google_analytics/google_analytics_measurement_protocol.py:51]: I think we could move the line 59 above the line 45, setting the property"t" with hit_type and eliminating the lines 62, 68, 76.

**Review by Mr-Lopes — CHANGES_REQUESTED** [review]
I've added a suggestion

File: megalista_dataflow/uploaders/google_analytics/google_analytics_measurement_protocol.py (hunk 1 above)
Comment (caiotomazelli) [reply megalista_dataflow/uploaders/google_analytics/google_analytics_measurement_protocol.py:51]: Nice catch, thanks!

**Review by Mr-Lopes — APPROVED** [review]
LGTM

**Pull request merged**